apps/src/sites/studio/pages/blocks/edit.js

```diff
@@ -123,6 +121,34 @@ function validateBlockConfig(editor) {
   }
 }
 
+// Ensure that the block config is valid JSON and meets other specific requirements.
+function validateJSON(editor) {
+  if (editor) {
+    // If the value isn't valid JSON, JSON.parse will throw
```

Comment:
This logic is all called from inside the `try` at line 110. If any error is thrown, it is caught and displayed in the `validationDiv`.